lib/codit/population/networks/home_locations.py

```diff
@@ -210,7 +213,8 @@ def build_households_home_list(district_type=DEFAULT_DISTRICT_TYPE):
         list_households_info += allocate_homes_to_district(pop_district[2] / AVERAGE_HOUSEHOLD_SIZE, tmp_coords_district)
 
     df_home_list = pd.DataFrame(list_households_info, columns=df_coordinates.columns)
-    df_home_list.to_csv(FULL_HOME_LIST_CSV, index=False)
```

Comment:
General comment (beyond the scope of this PR):
You should not have hardcoded paths in functions, this file path should be an argument or config.
I'll can add a PR to fix this in the future.

Reply:
Noted, thank you